lib/rules/named-grid-areas-no-invalid/index.js

```diff
@@ -25,7 +25,7 @@ const rule = (primary) => {
 			return;
 		}
 
-		root.walkDecls(/^grid-template-areas$/i, (decl) => {
+		root.walkDecls(/^grid$|^grid-template|$^grid-template-areas|$/i, (decl) => {
```

Comment:
[suggestion] Can we simplify this regex as below?

```suggestion
		root.walkDecls(/^(?:grid|grid-template|grid-template-areas)$/i, (decl) => {
```

Reply:
I had a feeling it could be better written as I was writing it :)